Gemfile

```diff
@@ -1,10 +1,12 @@
+# frozen_string_literal: true
+
```

Comment:
As long as no new gems are added, we can keep the reordering in this PR. Ideally though, a PR should be scoped to a single issue.